integration-tests/integration/bookings/amendBooking/amendBooking.spec.js

```diff
@@ -13,8 +13,8 @@ context('A user can amend a booking', () => {
     cy.task('reset')
     cy.task('stubLoginCourt', {})
     cy.login()
-    cy.task('stubUserEmail', 'ITAG_USER')
-    cy.task('stubUser', 'ITAG_USER', 'WWI')
+    cy.task('stubUserEmail', 'COURT_USER')
```

Comment:
renamed these as ITAG_USER is a specific nomis thing and these are actually auth users